scripts/populateInstances.js

```diff
@@ -35,9 +39,7 @@ const populateInstances = async (
       `cozy-stack instances token-cli ${domain} io.cozy.bank.operations`
     )
     execSync(
-      `ACH -u http://${domain} -y script banking/importFilteredOperations ${fixtureFile} ${
-        classes[i]
-      } ${operationsPerInstance} ${domain} -x -t ${ACHToken}`
+      `ACH -u http://${domain} -y script banking/importFilteredOperations ${fixtureFile} ${classes[i]} ${operationsPerInstance} ${domain} -x -t ${ACHToken}`
```

Comment:
Made me think that this is a requirement to have an ACH with this script. We should add ACH in the devDep and run this command with `yarn run ACH ...` (can be done in another PR)